.github/workflows/run-tutorials.yml

```diff
@@ -25,8 +25,21 @@ jobs:
           rm -rf /var/lib/apt/lists/*
       - name: Check FEniCSx version
         run: python3 -c "import dolfinx; print('FEniCS-X version '+dolfinx.__version__)"
-      - name: Install adapter
-        run:  pip3 install --user .
+      - name: Create venv for Dirichlet participants and install adapter
+        run: |
+          cd tutorials/partitioned-heat-conduction/dirichlet-fenicsx
+          python3 -m venv .venv
+          . .venv/bin/activate
+          pip3 install ../../..
```

Comment:
An alternative to this would be to add the venv setup in the `run.sh`, as we do in the rest of the Python-related tutorials. Any reason to do this in the test environment here?

Reply:
I followed this approach because we need to install the latest version of the adapter before running the tutorial. If we want to use the `run.sh` from the tutorial file we would have to somehow inject a pointer to the local version of the repo in the CI pipeline into the `requirements.txt` before executing the `run.sh` script. (not sure if this is even possible) I think it's easier and clearer to just prepare the `venv` outside of the `run.sh` script and then use the `run.sh` without modifications.

Reply:
I follow the same procedure here for the same reason: https://github.com/precice/fenics-adapter/pull/198/files

Reply:
Yes. The `run.sh` stays the same as always. Still needs some updates but we can do this when integrating into `precice/tutorials`